### PR TITLE
Created OpenAPIServiceModel

### PR DESCRIPTION
**Pull request by mndzup**
*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Review by Josh-Gearou — APPROVED** [review]

**Review by Josh-Gearou — CHANGES_REQUESTED** [review]
Looks like the build issue is a swift version issue (5.4).  You may need to debug the package.Swift file further or modify/update the travis config file.

File: Package.swift [hunk 1]
```diff
@@ -33,10 +33,14 @@ let package = Package(
...
             targets: ["SwaggerServiceModel"]),
+        .library(
+            name: "OpenAPIServiceModel",
+            targets: ["OpenAPIServiceModel"]),
     ],
     dependencies: [
-        .package(url: "https://github.com/tachyonics/SwaggerParser.git", from: "0.6.4"), 
+        .package(url: "https://github.com/tachyonics/SwaggerParser.git", from: "0.6.4"),
         .package(url: "https://github.com/jpsim/Yams.git", from: "4.0.0"),
+        .package(url: "https://github.com/Josh-Gearou/OpenAPIKit.git", from: "2.4.0"),
```
Comment (tachyonics) [inline Package.swift:43]: Is there a reason we are depending on Josh's fork of this dependency?

Comment (Josh-Gearou) [reply Package.swift:43]: No reason other than it looks like we forked SwaggerParser.  Also I assumed you did that so we could control changes we may want to make on our side.

Comment (mndzup) [reply Package.swift:43]: Is there a reason why the forked SwaggerParser would work as a dependency but the forked OpenAPIKit would not?

**Pull request closed without merging**